content/flux/v0.x/join-data/troubleshoot-joins.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Troubleshoot join operations
+description: >
+  Learn how to troubleshoot common behaviors and errors that may occur when using
+  the [`join` package](/flux/v0.x/stdlib/join).
+menu:
+  flux_0_x:
+    name: Troubleshoot joins
+    parent: Join data
+weight: 105
+---
+
+Learn how to troubleshoot common behaviors and errors that may occur when using
+the [`join` package](/flux/v0.x/stdlib/join).
+
+{{% note %}}
+#### Submit issues for unexplained behaviors or errors
+
+This is a "living" document that may be updated with common issues
+that users may run into when using the [`join` package](/flux/v0.x/stdlib/join)
+to join data. If you have questions about a behavior or error that is not
+documented here, please submit an issue on either the InfluxData Documentation
+or Flux GitHub repositories:
+
+- [Submit a documentation issue](https://github.com/influxdata/docs-v2/issues/new/choose)
+- [Submit a Flux issue](https://github.com/influxdata/flux/issues/new/choose)
+{{% /note %}}
+
+- [Troubleshoot join behaviors](#troubleshoot-join-behaviors)
```

Comment:
understand why you included, but may want to remove for simplicity until we have more behaviors?